src/presentation/http/router/note.ts

```diff
@@ -224,6 +224,14 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
 
     const addedNote = await noteService.addNote(content as JSON, userId as number, parentId); // "authRequired" policy ensures that userId is not null
 
+    /**
+     * Save note visit when note created
+     *
+     * @todo use even bus to save noteVisit
+     */
+    if (userId !== null) {
```

Comment:
Is the condition necessary? If we use authRequired in this route

Reply:
userId has number | null, type, so yes

Reply:
You can use `!` instead